_includes/section-title-box.html

```diff
@@ -3,6 +3,6 @@
 <header class="va-rr-sg-header va-rr-bg-{{templateclass}}">
```

Comment:
Add `usa-content` here, since the USWDS CSS adds a max width for `.usa-content p`.